### ISSUE TITLE
KeyError: wr.athena.read_sql_query()

### ISSUE THREAD

**Issue by nagomiso** [awslabs/aws-data-wrangler#159]
awswrangler version: 1.0.0

`wr.athena.read_sql_query()` raises a `KeyError` when I tried to execute following:

### sample.py

```sample.py
import awswrangler as wr

wr.athena.read_sql_query(
  "select * from companies limit 5",
  database="foo",   # dummy value
  workgroup="bar",  # dummy value
) 
```

### traceback

I do not encrypt in Amazon S3 so `EncryptionConfiguration` is seemed to be not set a value.

```python
Traceback (most recent call last):
  File "sample.py", line 6, in <module>
    workgroup="bar",
  File "/home/nagomiso/develop/.venv/lib/python3.7/site-packages/awswrangler/athena.py", line 408, in read_sql_query
    wg_s3_output, _, _ = _ensure_workgroup(session=session, workgroup=workgroup)
  File "/home/nagomiso/develop/.venv/lib/python3.7/site-packages/awswrangler/athena.py", line 546, in _ensure_workgroup
    wg_encryption: Optional[str] = config["EncryptionConfiguration"].get("EncryptionOption")
KeyError: 'EncryptionConfiguration'
```

### other library versions

```
boto3==1.12.39
botocore==1.15.39
```

**Comment by igorborgest** [comment]
Thank you @nagomiso! We will fix it ASAP.

**Comment by igorborgest** [comment]
@nagomiso fix available on version [1.0.1](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.0.1). Thanks!
